### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by SUPERCILEX**

### CODE REVIEW

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.10-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-3.1-all.zip
```
Comment (yshrsmz) [inline gradle/wrapper/gradle-wrapper.properties]: could you run `./gradlew wrapper` to update both `gradlew` and `gradlew.bat` ?

Comment (SUPERCILEX) [inline gradle/wrapper/gradle-wrapper.properties]: Done

**Comment by yshrsmz** [comment]
thanks a lot!

**Comment by SUPERCILEX** [comment]
You're welcome!

**Pull request merged**